kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/KyuubiCtlArguments.scala

```diff
@@ -128,7 +128,7 @@ class KyuubiCtlArguments(args: Seq[String], env: Map[String, String] = sys.env)
     val command = sys.env.getOrElse("_KYUUBI_CMD_USAGE",
```

Comment:
_KYUUBI_CMD_USAGE, is the first underscore a typo?

Reply:
emmm